src/IonParserBinaryRaw.ts

```diff
@@ -539,7 +539,7 @@ const MAX_BYTES_FOR_INT_IN_NUMBER = 6; // = floor(52 /* sig fig bits in number *
 const ZERO_POINT_ZERO = new Float64Array([0.0]);
 
 export class ParserBinaryRaw {
-  private buf = new ArrayBuffer(8);                   // TODO - what about threading?
+  private buf = new ArrayBuffer(8);                   // TODO - what about threading? no.
```

Comment:
If it's no longer a TODO, just remove the comment.

Reply:
done